### PR TITLE
Prepare for 0.13.0 release

### PR DESCRIPTION
**Pull request by rfmeier**
## Description

* Updated version from `0.12.1` to `0.13.0`.
* Update change log.

### CODE REVIEW

**Review by nickcernis — COMMENTED** [review]
@rfmeier Looking good!

Please could you update package-lock with the new version number too?

The way I do it is to:

1. Install a node version manager such as [fnm](https://github.com/Schniz/fnm).
2. Run `fnm install && fnm use` in the repo root to ensure you're using the correct version of Node.js (the version listed in the `.nvmrc` file in the repo root). Alternatively, you can manually check you're running the same version of Node.js.
3. Run `npm install` after bumping the version in `package.json`.
4. Commit the updated `package-lock.json` (it may have some other changes beyond the bumped package version, which is usually fine).

This ensures the lockfile is up to date with the release and doesn't change the next time someone pulls from main and then runs `npm install`.

**Review by nickcernis — DISMISSED** [review]

**Comment by mindctrl** [comment]
LGTM. Just noting we decided to release on Monday or Tuesday. Let's wait until then to merge. We may need to update the change log by then.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
 # Atlas Content Modeler Changelog
 
-## Unreleased
+## 0.13.0 - 2022-01-26
 ### Added
 - Opt-in anonymous usage tracking to help us make Atlas Content Modeler better (disabled by default).
```
Comment (anthonyburchell) [inline CHANGELOG.md:5]:
```suggestion
- Text Repeater Field - Added repeatable property to single and multi line text fields for returning arrays of publisher defined lists of strings.
- Opt-in anonymous usage tracking to help us make Atlas Content Modeler better (disabled by default).
```

File: readme.txt [hunk 2]
```diff
@@ -51,8 +51,15 @@ We recommend that you set Permalinks to a value other than “Plain” in your W
 ACM is primarily intended for headless WordPress applications. For that reason, the WordPress REST API and WPGraphQL are the only two officially supported APIs. That said, it is possible to fetch the data for your models in a traditional WordPress site by using the `rest_do_request()` PHP function that the REST API provides or the `graphql()` PHP function that WPGraphQL provides.
 = Where can I submit bug reports and feature requests? =
 You can submit feature requests and open bug reports in our [GitHub repo](https://github.com/wpengine/atlas-content-modeler).
+
 == Changelog ==
 
+= 0.13.0 - 2022-02-01 =
+
+* **Added:** Opt-in anonymous usage tracking to help us make Atlas Content Modeler better (disabled by default).
```
Comment (anthonyburchell) [inline readme.txt:59]:
```suggestion
* **Added:** Text Repeater Field - Added repeatable property to single and multi line text fields for returning arrays of publisher defined strings.
* **Added:** Opt-in anonymous usage tracking to help us make Atlas Content Modeler better (disabled by default).
```

**Review by anthonyburchell — APPROVED** [review]

**Pull request merged**